Makefile

```diff
@@ -36,7 +36,7 @@ protoc_abci:
 	## See https://stackoverflow.com/a/25518702
 	protoc $(INCLUDE) --gogo_out=plugins=grpc:. abci/types/*.proto
 	@echo "--> adding nolint declarations to protobuf generated files"
-	@awk '/package abci/types/ { print "//nolint: gas"; print; next }1' abci/types/types.pb.go > abci/types/types.pb.go.new
```

Comment:
Looks like this must have got merged in an non-working state since is no `package types`

Reply:
https://github.com/tendermint/tendermint/blob/257622cf6b1edd30ef7cdb30c3d0c2cb7b951562/abci/types/types.pb.go#L47

`@awk '/package types/ { print "//nolint: gas"; print; next }1' abci/types/types.pb.go > abci/types/types.pb.go.new` should work

Reply:
yeah that's what I changed it to, comment on wrong line